modules/ppcp-compat/resources/js/gzd-compat.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener(
+    'DOMContentLoaded',
+    () => {
+        const orderTrackingContainerId = "ppcp_order-tracking";
+        const orderTrackingContainerSelector = "#ppcp_order-tracking";
+        const gzdSaveButton = document.getElementById('order-shipments-save');
+        const loadLocation = location.href + " " + orderTrackingContainerSelector + ">*";
+
+        const disableTrackingFields = function () {
+            var childNodes = document.getElementById(orderTrackingContainerId).getElementsByTagName('*');
+            for (var node of childNodes) {
+                node.disabled = true;
+            }
+        }
+
+        const enableTrackingFields = function () {
```

Comment:
Maybe just create one function like `setEnabled...(bool)` to avoid duplication?

Reply:
done 0f6dcc8aa8ee4b44727f77156be79b139f8766f9